arb_os/arbsys.mini

```diff
@@ -437,52 +437,52 @@ impure func arbAddressTable_addressExists(topFrame: EvmCallFrame, calldata: Byte
         evmOp_return(0, 32);
     } else {
-        evmOp_revert_knownCodePc(address(102), 131, 0, 0);
+        evmOp_revert_knownCodePc(address(const::Address_ArbAddressTable), 131, 0, 0);
     }
 }
 
 impure func arbAddressTable_size(topFrame: EvmCallFrame, calldata: ByteArray) { // () -> uint
 // return the number of items in the address table
     if (bytearray_size(calldata) != 4) {
-        evmOp_revert_knownCodePc(address(102), 140, 0, 0);
+        evmOp_revert_knownCodePc(address(const::Address_ArbAddressTable), 140, 0, 0);
     }
     let size = addressTable_size(getGlobalAddressTable());
     if (evmCallStack_setTopFrameMemory(bytearray_set256(evmCallStack_getTopFrameMemoryOrDie(), 0, size))) {
         evmOp_return(0, 32);
     } else {
-        evmOp_revert_knownCodePc(address(102), 141, 0, 0);
+        evmOp_revert_knownCodePc(address(const::Address_ArbAddressTable), 141, 0, 0);
     }
 }
 
 impure func arbAddressTable_lookupIndex(topFrame: EvmCallFrame, calldata: ByteArray) {  // (uint index) -> address
 // return the address at slot index in the address table, or revert if index is beyond the end of the table
     if (bytearray_size(calldata) != 36) {
-        evmOp_revert_knownCodePc(address(102), 150, 0, 0);
+        evmOp_revert_knownCodePc(address(const::Address_ArbAddressTable), 150, 0, 0);
     }
     let index = bytearray_get256(calldata, 4);
     if let Some(addr) = addressTable_getByIndex(getGlobalAddressTable(), index) {
         if (evmCallStack_setTopFrameMemory(bytearray_set256(evmCallStack_getTopFrameMemoryOrDie(), 0, uint(addr)))) {
             evmOp_return(0, 32);
         } else {
-            evmOp_revert_knownCodePc(address(102), 151, 0, 0);
+            evmOp_revert_knownCodePc(address(const::Address_ArbAddressTable), 151, 0, 0);
         }
     } else {
-        evmOp_revert_knownCodePc(address(102), 152, 0, 0);
+        evmOp_revert_knownCodePc(address(const::Address_ArbAddressTable), 152, 0, 0);
     }
 }
 
 impure func arbAddressTable_decompress(topFrame: EvmCallFrame, calldata: ByteArray) {  // (bytes buf, uint offset) -> (address, uint)
     // read a compressed address from buf at offset, return (resulting address, updated offset)
     // revert if buf is too short
     if (bytearray_size(calldata) < 4+4*32) {
-        evmOp_revert_knownCodePc(address(102), 160, 0, 0);
+        evmOp_revert_knownCodePc(address(const::Address_ArbAddressTable), 160, 0, 0);
     }
     let offset = bytearray_get256(calldata, 36);
     let bytesLen = bytearray_get256(calldata, 68);
     if (offset >= bytesLen) {
-        evmOp_revert_knownCodePc(address(102), 161, 0, 0);  // buffer is too short
+        evmOp_revert_knownCodePc(address(const::Address_ArbAddressTable), 161, 0, 0);  // buffer is too short
     }
-    let stream = bytestream_new(bytearray_extract(calldata, 100+offset, bytesLen-offset));
+    let stream = bytestream_new(bytearray_extract(calldata, const::Address_ArbSys+offset, bytesLen-offset));
```

Comment:
This doesn't seem like a correct use of this constant

Reply:
Fixed